travis.sh

```diff
@@ -4,9 +4,26 @@ set -e
 export CXX=g++-9
 export CC=gcc-9
 
+
 # Add the current working directory to $PATH so that tests can find bedrock.
 export PATH=$PATH:`pwd`
 
+# Configure ccache settings for travis.
+export PATH=/usr/lib/ccache:$PATH
```

Comment:
Same discussion as the other PR if we can move this stuff to a config file.

Reply:
This is for travis on ly